src/model/encoding/__tests__/convertFromHTMLToContentBlocks-test.js

```diff
@@ -279,3 +279,33 @@ test('Should preserve entities for whitespace-only content', () => {
     experimentalTreeDataSupport: false,
   });
 });
+
+test('Should import recognised draft li depths', () => {
+  const html_string = `
+    <ul>
+      <li class="public-DraftStyleDefault-depth0">depth0</li>
+      <li class="public-DraftStyleDefault-depth1">depth1</li>
+      <li class="public-DraftStyleDefault-depth2">depth2</li>
+      <li class="public-DraftStyleDefault-depth3">depth3</li>
+      <li class="public-DraftStyleDefault-depth4">depth4</li>
+    </ul>
+  `;
+  assertConvertFromHTMLToContentBlocks(html_string, {
+    experimentalTreeDataSupport: false,
+  });
+});
+
+test('Should import recognised draft li depths when nesting enabled', () => {
```

Comment:
This is almost right - but I have one concern.

With "nesting enabled", the new features should create a block map where the "depth" property is not used, and instead we have true nesting set up using 'children' and 'parent' properties on the blocks.

See https://github.com/facebook/draft-js/blob/master/src/model/encoding/__tests__/__snapshots__/convertFromHTMLToContentBlocks-test.js.snap#L303 for an example.

It looks like with this change, we default to using the 'depth' property again for nesting, even with the 'nesting' flag enabled, when those classes are present.

I think we should add a conditional for your check that sets the 'depth' and disable it when the nesting flag is true.

Reply:
Ok - thanks for the pointer. So in the instance that `nesting` is `true` should there be an alternative way to handle converting the li indentation?